scripts/.env

```diff
@@ -7,3 +7,23 @@ export SIGNER2_MNEMONIC="night renew tonight dinner shaft scheme domain oppose e
 export SIGNER3_MNEMONIC="step endless survey brand topic warrior merry boat metal throw tag recycle pitch animal drill jar hero library arm swift kitten proof acoustic chef"
 export CRONOS_ADMIN="crc12luku6uxehhak02py4rcz65zu0swh7wjsrw0pp"
 export IBC_CRO_DENOM="ibc/6411AE2ADA1E73DB59DB151A8988F9B7D5E7E233D8414DB6817F8F1A01611F86"
+export USER1_MNEMONIC="ill hawk retreat wedding lobster decide recall public panel hint news snack excuse major glass obvious mad balance beef firm shiver century mean access"
```

Comment:
you don't have to hardcode the mnemonics, the pystarport can create random ones if it's not present.

Reply:
you mean fund these user later instead of provide for jsonnet?

Reply:
I mean let pystarport generate the mnemonics automatically instead of hardcode them, just remove the `mnemonic` field from the validator object.